vendor/bat-native-ledger/BUILD.gn

```diff
@@ -99,9 +99,14 @@ source_set("ledger") {
     "src/bignum.h",
     "src/ledger_impl.cc",
     "src/ledger_impl.h",
+    "src/transaction_info.cc",
```

Comment:
this should be inside `src/bat/ledger/ledger.cc`

Reply:
now that I am more thinking about it, let's just move it inside `src/bat/ledger/`. I will create PR that will split `src/bat/ledger/ledger.cc` into separate files, this way I think it will be easier to track things than in one big ledger.cc file

Reply:
https://github.com/brave/brave-browser/issues/3477

Reply:
👍 done